.github/workflows/ci.yaml

```diff
@@ -167,3 +168,14 @@ jobs:
       - name: Push Latest Package to Upbound
         if: env.XPKG_ACCESS_ID != ''
         run: "./crossplane --verbose xpkg push --package-files $(echo *.xpkg|tr ' ' ,) ${{ env.XPKG }}:latest"
```

Comment:
Note that I've intentionally omitted pushing a `:latest` tag to GHCR.

Crossplane packages shouldn't use the `:latest` tag. Package tags should always be semantic versions. This is required for the package manager to support dependency resolution. We need to update https://github.com/crossplane/crossplane/blob/main/contributing/specifications/xpkg.md to state this.